ide/idea/workspace.xml

```diff
@@ -193,7 +224,18 @@
     <configuration default="false" name="Cassandra" type="Application" factoryName="Application">
       <extension name="coverage" enabled="false" merge="false" sample_coverage="true" runner="idea" />
       <option name="MAIN_CLASS_NAME" value="org.apache.cassandra.service.CassandraDaemon" />
-      <option name="VM_PARAMETERS" value="-Dcassandra-foreground=yes -Dcassandra.config=file://$PROJECT_DIR$/conf/cassandra.yaml -Dcassandra.storagedir=$PROJECT_DIR$/data -Dlogback.configurationFile=file://$PROJECT_DIR$/conf/logback.xml -Dcassandra.logdir=$PROJECT_DIR$/data/logs -Djava.library.path=$PROJECT_DIR$/lib/sigar-bin -Dcassandra.jmx.local.port=7199 -ea -Xmx1G -Dcassandra.reads.thresholds.coordinator.defensive_checks_enabled=true -XX:HeapDumpPath=build/test" />
+      <option name="VM_PARAMETERS" value="
+                                          -Dcassandra-foreground=yes
+                                          -Dcassandra.config=file://$PROJECT_DIR$/conf/cassandra.yaml
+                                          -Dcassandra.jmx.local.port=7199
+                                          -Dcassandra.logdir=$PROJECT_DIR$/data/logs
+                                          -Dcassandra.reads.thresholds.coordinator.defensive_checks_enabled=true
+                                          -Dcassandra.storagedir=$PROJECT_DIR$/data
+                                          -Djava.library.path=$PROJECT_DIR$/lib/sigar-bin
+                                          -Dlogback.configurationFile=file://$PROJECT_DIR$/conf/logback.xml
+                                          -Xmx1G
+                                          -ea
```

Comment:
why did you remove `-XX:HeapDumpPath=build/test`?

Reply:
excellent eyes - mistake, will add back in.